README.md

```diff
@@ -134,77 +141,130 @@ ROOT
+|Test|Info,Warning|Period containment of registry entries|Tests whether the created and finalized dates of all registry entries in offentligJournal.xml are within the archival period specified in arkivuttrekk.xml, and whether the same registry entries (same systemIDs) can be found in arkivstruktur.xml.|
+
+**addml:**
+
+|Type|Results in|Title|
+|---|---|---|
+|Test|Info,Error|Validity of the arkivstruktur.xml checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the arkivstruktur.xsd checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the loependeJournal.xml checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the loependeJournal.xsd checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the offentligJournal.xml checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the offentligJournal.xsd checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the endringslogg.xml checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the endringslogg.xsd checksum specified in arkivuttrekk.xml|
 
 **endringslogg:**
-* [Info] Number of changes in endringslogg
-* [Test] Number of system IDs found in endringlogg, but not in arkivstruktur
\ No newline at end of file
+
+|Type|Results in|Title|Description|
+|---|---|---|---|
+|Check|Info|Change log entries|Provides information about the number of change log entries in endringslogg.xml.|
+|Test|Info,Warning|Change log references|Tests whether all objects referenced in the change log can be found by their systemID in arkivstruktur.xml.|
+
+## Reports
+The tool can produce one or more reports in different formats (see the examples above for more information on how to request multiple reports). Currently supported formats are Excel (pre-2007 binary format), Excel (post-2007 OOXML format), and XML. All reports include:
+* information about the execution:
+  * version of the tool
+  * list of key parameters specified for the execution
+* summary of the results
+* details for each executed validation rule (check or test), such as:
+  * title
+  * description
+  * info
+  * list of warnings
+  * list of errors
+
+Each list of warnings and errors aims to provide as detailed information as possible with regard to the exact location where an issue was encountered. Where applicable, this includes information such as system ID, file ID, case year, case number, record ID, record year, record number, line, or column.
+
+### Excel report
+The purpose of the Excel reports is to provide an intuitive human-readable representation of the validation outcome. It is color-coded and includes multiple links between generated sheets in order to facilitite easy navigation.
+
+### XML report
+The XML report, on the other hand, although not as intuitive to read, aims to provide a more long-lasting preservation format that is also machine-parsable. This is further guaranteed by the inclusion of an XSD schema with which it complies.
```

Comment:
`The XML report is not intended to be human-readable but to be used for long-term preservation of the validation results. The corresponding XSD schema is bundled with the report.`